{
    "results": [
        {
            "task_id": "instance-34.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-46.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-30.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-1.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-26.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-44.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-43.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-13.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-42.pddl",
            "success": false,
            "answer": "No answer found.",
            "token_usage": 2731,
            "num_times_pruned": 2
        },
        {
            "task_id": "instance-36.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-49.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-40.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-3.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-29.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-2.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-22.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-31.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-11.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-50.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-8.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-12.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-5.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-21.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-4.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-25.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-7.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-24.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-45.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-20.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-19.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-23.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-18.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-38.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-6.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-47.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-17.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-37.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-10.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-27.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-39.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-16.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-9.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-35.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-33.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-32.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-48.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-15.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-41.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-28.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        },
        {
            "task_id": "instance-14.pddl",
            "success": false,
            "answer": "Error occurred: TypeError: '>' not supported between instances of 'int' and 'coroutine'\nTraceback:\nTraceback (most recent call last):\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 236, in run_task_with_error_handling\n    return await self.run_task(tot, task, parallel, solution_given, breadth_first)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot_tester.py\", line 187, in run_task\n    answer = await tot.run(query, parallel=parallel, breadth_first=breadth_first, problem_width=problem_width)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 233, in run\n    await self._expand_depth_first(self.init_state, self.max_depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 303, in _expand_depth_first\n    await self._expand_node(state, depth=depth, parallel=parallel)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 282, in _expand_node\n    results.append(await t)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 265, in process_branch\n    succ = await self._generate_successor(node)\n  File \"/work/rleap1/leon.hamm/bachelor-thesis-dev/novelty_tot/tot.py\", line 438, in _generate_successor\n    prune = estimated_novelty == -1 or estimated_novelty > self.problem_width\nTypeError: '>' not supported between instances of 'int' and 'coroutine'\n",
            "token_usage": 0,
            "num_times_pruned": 0
        }
    ],
    "summary": {
        "total_tasks": 50,
        "successful_tasks": 0,
        "failed_tasks": 50,
        "total_token_usage": 2731,
        "total_num_times_pruned": 2,
        "num_early_stops": 0,
        "num_max_context_exceeded": 0,
        "num_max_retries_exceeded": 0,
        "num_retries": 0,
        "num_truncated_queries": 0,
        "num_shortened_prompts": 0
    },
    "duration": "0:00:23.946253"
}